.github/workflows/yaml-lint.yml

```
@@ -0,0 +1,36 @@
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Lint Workflow YAML
+        uses: ibiqlik/action-yamllint@v3
+        with:
+          file_or_dir: .github/workflows/yaml-lint.yml
+          config_file: .yamllint
+          format: github
+          no_warnings: true
+        if: ${{ !canceled() }}
```

actionlint

undefined function "canceled". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)